packages/nakama-js/web_socket_adapter.ts

```diff
@@ -145,23 +145,24 @@ export class WebSocketAdapterText implements WebSocketAdapter {
     }
 
     send(msg: any): void {
+        let payload = null;
```

Comment:
I wanted to confirm no functional changes here?

Reply:
None :) Just removed lines of code and made them reusable